### DISCUSSION TITLE
CDXML Parsing Issue

### DISCUSSION THREAD

**Discussion by X265065** [rdkit/rdkit#7467] [Development]
Hi Team,

I am trying to us below code where am reading a CDXML and printing the Molblock of it.
However, am getting the error for some fragments to be missing. Could you please help how can I fix it?

Code :-

from rdkit import Chem

mol = Chem.MolsFromCDXML(cdxml)
        
Error :-

[15:15:21] CDXMLParser: Schema 81 step 82 ReactionStepReactants fragment 73 not found in document.
[15:15:21] CDXMLParser: Schema 81 step 82 ReactionStepReactants fragment 42 not found in document.

CDXML is attached for your reference.
[ChemDraw.zip](https://github.com/rdkit/rdkit/files/15467398/ChemDraw.zip)

**Comment by bp-kelley** [comment]
This is a fun one.  These reactants are enclosed in groups which we don't support.  I'll file this as a bug as we should, at least, be able to get the mols out.

**Reply by X265065** [reply]
Hi Kelley,

Do we know when we will get a fix for this so we can get mols out atleast?
